ui/src/components/DashboardServices.js

```diff
@@ -0,0 +1,322 @@
+  }};
+`;
+
+const ClickableIcon = styled.i`
+  self-align: flex-end;
+`;
+
+const ServiceItem = ({
+  label,
+  status,
+  alerts,
+}: {
+  label: string,
+  status: string,
+  alerts: Alert[],
+}) => {
+  let date = null;
+
+  const getStartDate = (alerts: Alert[]) => {
+    // Check we only have alerts and set startsAt properties
+    const filtered = alerts.filter((item) => item && item.startsAt);
+    // Sort to make sure we have oldest alert first
+    filtered.sort((a, b) => new Date(a.startsAt) - new Date(b.startsAt));
+
+    if (filtered[0] && filtered[0].startsAt) return filtered[0].startsAt;
+    return '';
+  };
+
+  if (alerts.length) date = new Date(getStartDate(alerts));
+
+  if (!alerts.length && status === STATUS_HEALTH)
+    return (
+      <ServiceItemElement aria-label={label}>
+        <ServiceItemLabelWrapper>
+          <CircleStatus status={status} />
+          <ServiceItemLabel>{label}</ServiceItemLabel>
+        </ServiceItemLabelWrapper>
+      </ServiceItemElement>
+    );
+  else
+    return (
+      <NonHealthyServiceItemElement>
+        <Tooltip
+          // Right placement to avoid Z index issues between left sidebar and tooltip or out of screen tooltip displays
+          placement="top"
+          overlayStyle={{
+            // em sizing to handle font-size change on large screens
+            width: '20em',
+            height: '100%',
+          }}
+          overlay={
+            <NonHealthyPopUp>
+              <NonHealthyPopUpTitle>View details</NonHealthyPopUpTitle>
+              <NonHealthyPopUpItem>
+                <label>Severity</label>
+                <NonHealthyPopUpSeverity status={status}>
+                  {status}
+                </NonHealthyPopUpSeverity>
+              </NonHealthyPopUpItem>
+              {
+                // Checking date validity
+                date instanceof Date && !isNaN(date) && (
+                  <NonHealthyPopUpItem>
+                    <label>Start</label>
+                    <div>
+                      {`${date.toISOString().split('T')[0]}
+              ${date.toISOString().split('T')[1].slice(0, 8)}`}
+                    </div>
+                  </NonHealthyPopUpItem>
+                )
+              }
+            </NonHealthyPopUp>
+          }
+        >
+          <Link to="/alerts" data-testid="alert-link">
+            <ClickableServiceItemElement aria-label={label}>
+              <ServiceItemLabelWrapper>
+                <CircleStatus status={status} />
+                <ServiceItemLabel>{label}</ServiceItemLabel>
+              </ServiceItemLabelWrapper>
+              <ClickableIcon className="fas fa-angle-right" />
+            </ClickableServiceItemElement>
+          </Link>
+        </Tooltip>
+      </NonHealthyServiceItemElement>
+    );
+};
+
+const DashboardServices = () => {
+  const intl = useIntl();
+  const alertsLibrary = useAlertLibrary();
+
+  // K8s Master
+  const k8sAlerts = useHighestSeverityAlerts(
```

Comment:
In this specific case, we only gather two alerts: "K8sMasterServicesAtRisk" and "K8sMasterServicesDegraded"
and then we filter out the lower severity, so in the end, we should have at most one alert or an empty array. 
```suggestion
  const k8sHighestSeverityAlert = useHighestSeverityAlerts(
```